tiledb/sm/subarray/subarray.h

```diff
@@ -638,6 +638,12 @@ class Subarray {
   std::unordered_map<std::string, MemorySize> get_max_mem_size_map(
       ThreadPool* compute_tp);
 
+  /**
+   * Return relevant fragments as computed
+   * @return
+   */
+  const std::vector<unsigned>& relevant_fragments() const;
```

Comment:
Don't return a const-ref here. Just do `std::vector<unsigned>`.

Reply:
Why is that? I think that's consistent to the way we return member vectors.